src/styled/nav-bar.ts

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+import { colorMap } from './theme';
+
+const NavbarWrapper = styled('div')`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100%;
+    position: relative;
+    transform: translateZ(0);
+    z-index: 2;
```

Comment:
We should consider creating constants for the levels of layers we want to support such as 

```
$zindex-dropdown:       1000;
$zindex-sticky:         1020;
$zindex-fixed:          1030;
$zindex-modal-backdrop: 1040;
$zindex-modal:          1050;
$zindex-popover:        1060;
$zindex-tooltip:        1070;
```

Article here https://www.smashingmagazine.com/2021/02/css-z-index-large-projects/